src/components/Navbar/TopNavbar.tsx

```diff
@@ -1,40 +1,119 @@
-import React from "react";
-import { Navbar, Nav } from "react-bootstrap";
+import React, { useState } from "react";
 import "./TopNavbar.scss";
-import { Link } from "react-router-dom";
+import { Link, NavLink } from "react-router-dom";
 import Logo from "../../assets/logo.svg";
+import loggedProfile from "../../assets/man.png";
+import UnknownProfile from "../../assets/profile.svg";
+import { Button } from "react-bootstrap";
 
 function TopNavbar() {
+  const [isLoggedin, setisLoggedin] = useState(() => {
```

Comment:
Camel casing. It should be setIsLoggedIn and isLoggedIn

Reply:
changing in topnav bar PR